mantle/platform/conf/conf.go

```diff
@@ -181,75 +161,21 @@ func (u *UserData) IsIgnitionCompatible() bool {
 
 // Render parses userdata and returns a new Conf. It returns an error if the
 // userdata can't be parsed.
-func (u *UserData) Render(ignv2 bool) (*Conf, error) {
-	return u.RenderForCtPlatform(ignv2, "")
+func (u *UserData) Render() (*Conf, error) {
+	return u.RenderForCtPlatform("")
 }
 
-// RenderForCtPlatform parses userdata and returns a new Conf. It returns an error if the
-// userdata can't be parsed.
-func (u *UserData) RenderForCtPlatform(ignv2 bool, ctPlatform string) (*Conf, error) {
+// RenderForCtPlatform parses userdata and returns a new Conf. It returns an
+// error if the userdata can't be parsed.
+func (u *UserData) RenderForCtPlatform(ctPlatform string) (*Conf, error) {
```

Comment:
`ctPlatform` is now unused.  CLCT expected the platform name as a parameter and produced platform-specific Ignition configs, but Butane doesn't do that.  So, this function should be renamed and its parameter unplumbed from the call chain.  That should let us get rid of the `github.com/coreos/container-linux-config-transpiler` import throughout the codebase.